### ISSUE TITLE
Vybete všechny možnosti, které definují graf G=(V,E)  jako strom (ekvivalentní k definici). Tj. vyberte taková tvrzení, že každý strom splňuje příslušné tvrzení a každý graf, který splňuje příslušné tvrzení je strom.

### ISSUE THREAD

**Issue by MSKL** [MSKL/fit-bi-ag1-rozstrel#19]
<img width="873" alt="vybete vsechny moznosti ktere definuji graf g v e jako strom ekvivalentni k definici tj vyberte takova tvrzeni ze kazdy strom splnuje prislusne tvrzeni a kazdy graf ktery splnuje prislusne tvrzeni je strom" src="https://user-images.githubusercontent.com/9153692/35191011-76c66870-fe70-11e7-8963-f9dabef15d7a.png">
Asi to poslední? Myslel jsem, že to nesplňuje souvislost.

**Comment by MSKL** [comment]
<img width="863" alt="vybete vsechny moznosti ktere definuji graf g v e jako strom" src="https://user-images.githubusercontent.com/9153692/35201972-bd764a60-ff1f-11e7-9051-cb8aa7bc68e5.png">

**Comment by jstorm31** [comment]
Jo, asi tak jak píšeš. Taky mě zmátlo, že tam není souvislý. Ale možná se nějak implicitně bere, že ty vrcholy _u, v_ můžou být libovolné a když je mezi libovolnými dvěma vrcholy cesta, tak je graf souvislý... nevím, no 😄.
